mlos_bench/mlos_bench/run.py

```diff
@@ -84,8 +84,8 @@ def _optimize(env: Environment,
         # Load (tunable values, benchmark scores) to warm-up the optimizer.
         # `.load()` returns data from ALL merged-in experiments and attempts
         # to impute the missing tunable values.
-        (configs, scores) = exp.load()
-        opt.bulk_register(configs, scores)
+        (configs, scores, status) = exp.load()
+        opt.bulk_register(configs, scores, status)
```

Comment:
I think there's a TODO somewhere in `mlos_core_optimizer.py` to handle FAILED status that we should maybe mark down if appropriate.

Reply:
yep, thank you for reminding me. fixed that, too!